docs/Concepts/Stores.md

```diff
@@ -54,7 +57,7 @@ Use the [`/keys`](https://consensys.github.io/quorum-key-manager/#tag/Keys) REST
 
 An Ethereum store manages Ethereum accounts and performs Ethereum-related crypto-operations (for example, signing transactions).
 
-An Ethereum store can generate and import accounts but does not allow access to the private key of any account.
+An Ethereum store can generate and import accounts but does not allow to expose the private key of any account.
```

Comment:
I prefered the "access to" phrase here. "Does not allow to expose" is not correct english I think, should be "does not allow exposing"

Reply:
updated